c7n/output.py

```diff
@@ -192,10 +198,8 @@ def compress(self):
                         shutil.copyfileobj(sfh, zfh, length=2**15)
                     os.remove(fp)
 
-    def use_s3(self):
-        raise NotImplementedError()  # pragma: no cover
-
 
+@blob_outputs.register('fs')
```

Comment:
imo it would be better to use `file` to signify local disk output as it's a more familiar URI scheme and per [RFC-8089](https://tools.ietf.org/html/rfc8089)